models/jachin_bmk.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import (Integer, Column, String, Date)
+
+from models.config import Base, db_engine
+
+
+class BmkTable(Base):
+    __tablename__ = 'bmk_table'
+    __table_args__ = {'extend_existing': True}
+    bmk_id = Column('bmk_id', Integer, primary_key=True)
+    bmk_name = Column('bookmark_name', String(50))
+    bmk_url = Column('bookmark_url', String(200))
```

Comment:
Maybe we do not want a URL and Title to be empty 
So I guess setting `nullable=False` in column definition